.github/workflows/publish_to_dockerhub.yml

```diff
@@ -23,8 +25,21 @@ jobs:
           username: ${{ env.DOCKER_USER }}
           password: ${{ env.DOCKER_TOKEN }}
 
-      - name: Docker Build
-        run: make docker-build
+      - name: Install Dev Requirements
+        run: pip install -r dev-requirements.txt
 
-      - name: Docker Push
-        run: make docker-push
+      - name: Build Image
+        run: nox -s "build(prod)"
+
+      - name: Push Dev Tag
+        run: nox -s "push(dev)"
+
+      - name: Check Prod Tag
+        id: check-tag
+        run: |
+          if [[ ${{ github.event.ref }} =~ ^refs/tags/[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+              echo ::set-output name=match::true
+          fi
+      - name: Push Prod Tags
```

Comment:
Will this publish the `latest` tag in addition to the tag number?

Reply:
yep! this is copy/pasta from fidesctl so you can see what the output looks like here: https://hub.docker.com/r/ethyca/fidesctl/tags